data/ext/bio/Workflow.rdfa

```diff
@@ -0,0 +1,53 @@
+<div>
+  <!--
+  # Schema.org Bioschemas extension proposal
+  * | Project: https://bioschemas.org
+  * | MailingList: https://lists.w3.org/Archives/Public/public-bioschemas/
+  * | Issues:
+  -->
+
+  <div typeof="rdfs:Class" resource="http://schema.org/Workflow">
+      <link property="http://schema.org/isPartOf" href="http://bio.schema.org" />
+      <span class="h" property="rdfs:label">Workflow</span>
+      <span property="rdfs:comment">"A workflow consists of an orchestrated and repeatable pattern of activity enabled by the systematic organization of resources into processes that transform materials, provide services, or process information. It can be depicted as a sequence of operations, the work of a person or group, the work of an organization of staff, or one or more simple or complex mechanisms. (source Wikipedia.org)"</span>
+      <span>Subclass of: <a property="rdfs:subClassOf" href="http://schema.org/SoftwareSourceCode">SoftwareSourceCode</a></span>
+      <span>Source:  <a property="dc:source" href="http://bioschemas.org">Bioschemas</a></span>
+  </div>
+
+  <div typeof="rdf:Property" resource="http://schema.org/softwareRequirements">
```

Comment:
What about hardware/materials requirements? From the broad workflow definition, I would say baking a cake fits in there (to bake a cake a follow a organize resources (ingredients) and following a process, sequence of steps, so I can transform those materials, input, into a cake, output ). It is possible I am missing something here but maybe the workflow definition needs to be narrow down. Could programmatic workflow maybe work here?

Reply:
This touches on other kind of workflows that are not computational, and thus do not have formal parameters. Even in Bio* world there is lots of confusion on this, as people have lab protocols called workflows - more like business workflows and even what Wikipedia just call <https://en.wikipedia.org/wiki/Workflow>

Such workflows are presumably not really subclasses of `SoftwareSourceCode` (unless formalized in BPM etc). If we imagine someone comes along and want to describe these they may found our `Workflow` in the way, and a name like `ComputationalWorkflow` make more sense. 

However we want to avoid duality as both might have inputs and outputs and steps - would a future `Workflow` be possible to inject later as a new second parent of `ComputationalWorkflow`?

Reply:
ComputationalWorkflow is a better option, indeed. And, yes, a future generic Workflow type could become parent of ComputationalWorkflow. Hard to know whether a generic Workflow type will come later so model your ComputationalWorkflow on its own. Maybe also worth to see the HowTo class in schema.org, that one allows to describe things that can be split into steps (like a how to section), there could be some things there that could be useful for  ComputationalWorkflow as well, or you could decide to inherit from it.